docs/pages/world/account-delegation.mdx

```diff
@@ -33,6 +34,72 @@ With MUD delegation, you keep your existing account, but (temporarily) approve o
 
 The most common type of delegation is when a delegator address allows a specific delegatee address to act on its behalf.
 
+### Delegation types
+
+Delegation can either be unlimited or limited by various factors (time, calldata, etc.).
+To create a limited delegation you need a contract that implements the [`DelegationControl`](https://github.com/latticexyz/mud/blob/main/packages/world/src/DelegationControl.sol) interface to decide whether or not to approve a specific call.
+The owner of the root namespace (and therefore the `World`) needs to deploy the [`StandardDelegationModule`](https://github.com/latticexyz/mud/blob/main/packages/world-modules/src/modules/std-delegations/StandardDelegationsModule.sol), which includes three delegation control contracts:
```

Comment:
This sounds like the owner needs to deploy the standard delegations module to be able to use limited delegations in general. Maybe slightly rephrase to sth like this:

```suggestion
The [`StandardDelegationModule`](https://github.com/latticexyz/mud/blob/main/packages/world-modules/src/modules/std-delegations/StandardDelegationsModule.sol) includes three limited delegation control contracts. To use them, the owner of the root namespace needs to install this module.
```